bundle-workflow/scripts/default/opensearch-dashboards/build.sh

```diff
@@ -68,3 +68,4 @@ echo "BUILD RELEASE ZIP FOR $PLUGIN_NAME"
 (cd ../OpenSearch-Dashboards/plugins/$PLUGIN_NAME && yarn plugin-helpers build)
 echo "COPY $PLUGIN_NAME.zip"
 cp -r ../OpenSearch-Dashboards/plugins/$PLUGIN_NAME/build/$PLUGIN_NAME-$VERSION.zip $OUTPUT/plugins/
+rm -rf ../OpenSearch-Dashboards/plugins/$PLUGIN_NAME
```

Comment:
Why is this needed?

Reply:
without this I observe error below during plugin build. Synced with rocky, seems we need to delete the plugin folder content after the artifact is generated, otherwise it will impact the following plugin build.
```
    The conflicting dependencies are:

        cypress
          ^6.0.0 => opensearch-alerting-dashboards
          ^5.0.0 => anomaly-detection-dashboards
        formik
          ^2.2.6 => opensearch-alerting-dashboards
          ^2.2.5 => anomaly-detection-dashboards
```

Reply:
I think we need an issue to make these builds more independent. I am not sure how this would work, but reading this it seems like an _installation_ of a plugin pollutes another plugin installation?

Reply:
yes one pollutes another. I will create an issue to track this.